dbms/src/Client/Benchmark.cpp

```diff
@@ -206,6 +209,18 @@ class Benchmark
 				report(info_per_interval);
 				watch.restart();
 			}
+
+			if (num_repetions > 0 && info_total.queries >= num_repetions)
+			{
+				std::cout << "The execution is broken since request number of loops is reached\n";
```

Comment:
In fact, iterations, not loops.
You may pass many queries to benchmark and it will loop many times through them.